index.html

```diff
@@ -1,23 +1,79 @@
 <html>
 	<head>
-		<title>GitGitGadget - send GitHub PRs to the Git mailing list</title>
+		<title>GitGitGadget - contributing git.git patches via GitHub PRs</title>
+		<style>
+			body {
+				font-family: sans-serif;
+				background-color: #d7cdc0ff;
+				background-image: linear-gradient(#d7cdc0ff, #fdf7f6ff);
+				margin-right: 10%;
+				margin-left: 10%;
+			}
+			.block {
+				background-color: rgb(247, 243, 239);
+				color: #000000;
+				padding: 1em;
+				padding-top: .1em;
+				padding-bottom: .1em;
+				border-radius: 2em;
+			}
+			.banner {
+				text-align: center;
+				font-weight: bold;
+			}
+			big {
+				font-size: 400%;
+			}
+			tiny {
+				font-size: 70%;
+			}
+		</style>
 	</head>
 	<body>
-		<h1>GitGitGadget - send GitHub PRs to the Git mailing list</h1>
+		<div class="banner">
+			<img src="images/gitgitgadget-explanation.png" />
+			<big>GitGitGadget</big><br />
+			Contributing git.git patches via GitHub PRs<br />
+			<tiny>... one iteration at a time... </tiny>
+		</div><hr />
 
-		<p>
-		This project intends to help submit patch series to the Git mailing list. The process is not quite as painless for the contributor as opening Pull Requests, but at least it is much less painful than having to perform all the steps manually (see e.g. <a href="https://github.com/git-for-windows/git/blob/master/CONTRIBUTING.md">Git for Windows' detailed instructions how to contribute patches to the Git project</a>).
-		</p>
+		<div class="block">
+			<p>
+				Unlike most open source projects, Git itself does not accept code contributions via Pull Requests. Instead, patches are <a href="https://github.com/git/git/blob/master/Documentation/SubmittingPatches">submitted to the Git mailing list</a> for review and will be applied manually by the Git maintainer.
```

Comment:
It's difficult to review this one-line-per-paragraph text.

Reply:
I wrapped at the end-of-sentence markers.